cmd/app.go

```diff
@@ -16,6 +16,7 @@ import (
 
 	"sigs.k8s.io/cloud-provider-kind/pkg/config"
 	"sigs.k8s.io/cloud-provider-kind/pkg/controller"
+	usedImage "sigs.k8s.io/cloud-provider-kind/pkg/image"
```

Comment:
Note: import aliases should be all lower case.

Reply:
@stmcginnis  Thank you for pointing that out. I’ve committed the updated changes. Please feel free to let me know if there’s anything else that needs to be addressed!